### DISCUSSION TITLE
ComparisonError: Cluster level Namespace "helm-guestbook" can not be managed when in namespaced mode

### DISCUSSION THREAD

**Discussion by karl18** [argoproj/argo-cd#12488] [Q&A] [answered]
Hi, 
Since I deployed Argocd and configured all RBAC and ConfigMap to access all namespaces by adding `application.namespaces: '*'`, I still cannot any deploy any Application in any of the namespaces except for the same namespace where ArgoCD installed `argocd` namespace.

I see Sync errors on the ArgoCD Dashboard as follows:
<img width="1149" alt="Screenshot 2023-02-15 at 09 33 59" src="https://user-images.githubusercontent.com/30005437/218962094-a6a665dd-c110-40c9-814d-a1ef78abf9a5.png">

 
<img width="1252" alt="Screenshot 2023-02-15 at 09 33 52" src="https://user-images.githubusercontent.com/30005437/218962113-0ff04b07-d783-40e2-bcaf-9ff29e24ee63.png">

So I see the same error on all namespaces, either I **manually create them** by adding some labels such as: 
`argocd.argoproj.io/managed-by: argocd` or tick the checkbox for `AUTO-CREATE NAMESPACE` I everytime see this sync error: 
`Unable to load data: error getting cached app state: ComparisonError: Cluster level Namespace "helm-guestbook" can not be managed when in namespaced mode`

Anyone could give a hand here ? 
Appreciate it!

**Comment by karl18** [comment, marked as answer]
solved by removing namespace name in the **cluster settings** in arogcd UI 
<img width="1257" alt="Screenshot 2023-02-16 at 13 31 34" src="https://user-images.githubusercontent.com/30005437/219363236-61d28f1d-78bf-4f54-8328-fedc97760211.png">
<img width="858" alt="Screenshot 2023-02-16 at 13 31 42" src="https://user-images.githubusercontent.com/30005437/219363246-e84a3e81-f921-4727-a0ba-eea981aaf414.png">

**Comment by EBCorrea** [comment]
Hi, I had the same issue, that's the true solution! thank you so much :)
